src/sidebar/components/group-list.js

```diff
@@ -66,6 +95,17 @@ function GroupListController(
     groups.focus(groupId);
   };
 
+  /**
+   * This is a quick and dirty hack to show styling, and needs refactoring.
+   */
+  this.toggleGroupDetails = function() {
```

Comment:
This expands the wrong group if there are multiple out-of-scope groups. Given this setup:

<img width="365" alt="Screenshot 2019-03-14 14 02 57" src="https://user-images.githubusercontent.com/2458/54362984-e50b3600-4661-11e9-873e-672ce68ffe69.png">

Clicking on the "Test restricted group" will toggle the top one.

What you need to do instead is record the expanded state (eg. in a local variable in the `GroupListController` function) and then query that in the template to decide whether to add the `expanded` class to a particular entry.